docs/UserGuide.md

```diff
@@ -218,9 +229,23 @@ _Details coming soon ..._
 
 ## FAQ
 
+Listed here are some frequently asked questions. Should you require any other assistance, feel free to [contact us](#contact-us).
+
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous GuestBook home folder.
 
+**Q**: Will my data be automatically saved?<br>
+**A**: Yes, Guest Book automatically saves your data after every command entered.
```

Comment:
Should we use GuestBook rather than Guest Book?

Reply:
Agreed.